heltour/tournament/admin.py

```diff
@@ -30,6 +30,8 @@
 from django.contrib.staticfiles.templatetags.staticfiles import static
 from django.db.models.signals import post_save
 from django.dispatch.dispatcher import receiver
+from heltour.tournament.team_rating_utils import team_rating_range, team_rating_variance
+from heltour.tournament.create_teams import make_league, total_happiness, reduce_variance
```

Comment:
Since the purpose of these methods isn't obvious from their names, I would prefer them to be at least partially qualified

Reply:
make_league specifically is a bad name. should be `generate_teams` or something like that.  I named it make league because it doesn't just create the teams it also assigns all the alternates.  But the word league has other meanings here.